anitailango/affinity
# Add basic Chrome extension examples

To test:
- Navigate to `chrome://extensions` in Chrome, make sure Developer Mode is on
- Click "Load unpacked"
- Click on the folder "chrome-extension" 
(To test the React.js version, use the folder "chrome-extension-react/build")
